user/details/script.js

```diff
@@ -661,6 +659,46 @@ function showContent() {
   section3.classList.remove('hide');
 }
 
+function ifUserNotSuperUser() {
```

Comment:
Function name can be better here, write its name according to the action/usage

Reply:
done